packages/flutter/test/material/dropdown_menu_test.dart

```diff
@@ -125,7 +125,7 @@ void main() {
 
     final Finder textField = find.byType(TextField);
     final Size anchorSize = tester.getSize(textField);
-    expect(anchorSize, const Size(180.0, 54.0));
+    expect(anchorSize, const Size(180.0, 56.0));
```

Comment:
Why did this change?

Reply:
I changed the padding style for the trailing icon button from EdgeInsets.symmetric(horizontal: 4.0) to EdgeInsets.all(4.0) for some vertical space, so the height of the text field gets changed.